core-plugins/src/test/java/co/cask/hydrator/plugin/transform/StructuredtoAvroTest.java

```diff
@@ -72,4 +74,16 @@ public void testOutputSchemaUsage() throws Exception {
     Assert.assertNull(result.get("age"));
   }
 
+  @Test
+  public void testByteArrayConversionToByteBuffer() throws Exception {
+    Schema outputSchema = Schema.recordOf("output",
+                                          Schema.Field.of("testForBytes", Schema.of(Schema.Type.BYTES)));
```

Comment:
@russorat  Added code to handle nullable field and corresponding unit test.Please review